### PR TITLE
update based on customer query

### PR DESCRIPTION
**Pull request by anwara-ms**
Added a section on what happens when a message is edited and retention is in place.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/retention-policies.md [hunk 1]
```diff
@@ -60,6 +60,14 @@ To create a retention policy for Teams chats and channel messages, use the instr
 
 That page has additional information about creating and managing retention policies for other workloads in Microsoft 365. For example, you might want to also create a retention policy for Microsoft 365 Groups to retain and delete files that are accessed in Teams and stored in OneDrive or SharePoint.  
 
+## Retention of edited messages
+
+Retention on Teams messages works on the timestamp of creation of original message. Lets take a scenario- If there is a policy of 30-day deletion on Teams chat or channel messages and a user sends a message (“Original message”) on day -1, later modifies it to “edited message”, on day 15.
```
Comment (JohanFreelancer9) [inline Teams/retention-policies.md:65]:
```suggestion
Retention on Teams messages works on the timestamp of the creation of the original message. For example, in a scenario where there is a 30-day deletion policy on Teams chat or channel messages and a user sends a message (“Original message”) on day -1, then later modifies it to an “edited message” on day 15.
```

File: Teams/retention-policies.md [hunk 2]
```diff
@@ -60,6 +60,14 @@ To create a retention policy for Teams chats and channel messages, use the instr
...
+
+Here, the original message will be copied to a temporary location in the backend storage and deleted within 21 days. If there is Legal hold (via eDiscovery portal) or Retention hold in place on the user, then all versions of original, deleted and edited messages are stored as long as the hold policy is active.
```
Comment (JohanFreelancer9) [inline Teams/retention-policies.md:67]:
```suggestion
In this scenario, the original message will be copied to a temporary location in the backend storage and deleted within 21 days. If there is a Legal hold (via the eDiscovery portal) or a Retention hold in place on the user, then all versions of original, deleted, and edited messages are stored as long as the hold policy is active.
```

File: Teams/retention-policies.md [hunk 3]
```diff
@@ -60,6 +60,14 @@ To create a retention policy for Teams chats and channel messages, use the instr
...
+
+The “edited message” would be deleted by the retention policy on Day-30 i.e. based on the timestamp on the original message.
```
Comment (JohanFreelancer9) [inline Teams/retention-policies.md:69]:
```suggestion
The edited message would be deleted by the retention policy on day 30, that is, based on the timestamp on the original message.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestions.

**Comment by anwara-ms** [comment]
can we publish plz

**Comment by scanum** [comment]
@cabailey PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by cabailey** [comment]
@scanum I see a merge conflict that need to be resolved

**Comment by cabailey** [comment]
Closing this because I've updated this page to include concepts for edited messages (also added won't be deleted if hold also exists) & in the process of updating the M365 page where we go into detail about timings and what happens when messages are edited or deleted. I'm also looking to add a walkthrough example like this on that page, where it will be clearer where the 21 days comes from.

**Comment by anwara-ms** [comment]
Hi Carol
Can you please tell me what exactly has been updated re: edited messages? The customer confusion was specifically about timelines of edited messages and I believe that is not addressed.

Please, I want the original message to be posted to Teams documentation. Roberto started a thread on this 21 day delay and he got clarification from chatservice. I'll fwd that msg as FYI to you.


  --

Anwar | PM @ Teams  IP + Compliance | LinkedIn<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.linkedin.com%2Fin%2Fanwarali&data=04%7C01%7CAli.Anwar%40microsoft.com%7C61a24ee754db4b1493e408d6b4dc7b41%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636895256392090042%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C-1&sdata=KpQ4z1jxZOc8s8IWXAYS3oVAbqKpT94E%2BzEB%2FLypv2c%3D&reserved=0>



________________________________

**Comment by cabailey** [comment]
Continue conversation internally.

**Pull request closed without merging**